docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -35,17 +35,17 @@ The main features of the Sensor API include:
 
   **Table: Supported sensor types**
 
-  | Supported sensor types                   |                                          |                                          |
-  |------------------------------------------|------------------------------------------|------------------------------------------|
-  | [Accelerometer](#accelerometer)          | [Heart Rate Monitor Sensor](#hrm)        | [Proximity Sensor](#proximity)           |
-  | [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Humidity Sensor](#humidity)             | [Rotation Vector Sensor](#rotation)      |
-  | [Gravity Sensor](#gravity)               | [Light Sensor](#light)                   | [Significant Motion Sensor](#significant_motion) |
-  | [Gyroscope](#gyro)                       | [Linear Acceleration Sensor](#lin_accelerometer) | [Sleep Monitor](#sleep_monitor)          |
-  | [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Magnetic Sensor](#magnetic)             | [Temperature Sensor](#temperature)       |
-  | [Heart Rate Monitor LED Green Sensor](#hrm_green) | [Orientation Sensor](#orientation)       | [Ultraviolet Sensor](#ultraviolet)       |
-  | [Heart Rate Monitor LED IR Sensor](#hrm_ir) | [Pedometer](#pedometer)                  | [Uncalibrated Gyroscope](#uncal_gyro)    |
-  | [Heart Rate Monitor LED Red Sensor](#hrm_red) | [Pressure Sensor](#pressure)             | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-
+  | Supported sensor types                                        |                                                  |                                                  |
```

Comment:
Can we keep either of the one?
Both represent the same thing.
What do you say?

**Table: Supported sensor types**

 Supported sensor types                                        |                                                  |                                                  |

Reply:
I think that It looks good to remove the first row of the table(Supported sensor types | | |).

Reply:
@bbrto21 thanks for your suggestion.